clustered/client/src/main/java/org/ehcache/clustered/client/internal/service/DefaultClusteringService.java

```diff
@@ -163,6 +163,20 @@ public void start(final ServiceProvider<Service> serviceProvider) {
     }
   }
 
+  @Override
+  public void startForMaintenance(ServiceProvider<MaintainableService> serviceProvider, MaintenanceScope maintenanceScope) {
```

Comment:
I moved it because it was annoying to go back and forth between them. And the private methods used by them are now under them